### PR TITLE
Rename "community" label to "help wanted" as suggested by Github

### PR DESCRIPTION
**Pull request by jrudolph**
Also link to CoC on our README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ You can join these groups and chats to discuss and ask Akka related questions:
 - Q&A: [![stackoverflow: #akka-http][stackoverflow-badge]][stackoverflow]
 - Issue tracker: [![github: akka/akka-http][github-issues-badge]][github-issues]
 
+All of our forums, chat rooms, and issue trackers are governed by our [Code Of Conduct](CODE_OF_CONDUCT.md).
```
Comment (ktoso) [inline README.md]: nice phrasing

File: README.md [hunk 2]
```diff
@@ -59,7 +61,7 @@ Contributing
 Contributions are *very* welcome!
 
 If you see an issue that you'd like to see fixed, the best way to make it happen is to help out by submitting a pull request.
-For ideas of where to contribute, [tickets marked as "community"](https://github.com/akka/akka-http/issues?q=is%3Aissue+is%3Aopen+label%3Acommunity) are a good starting point.
+For ideas of where to contribute, [tickets marked as "help wanted"](https://github.com/akka/akka-http/labels/help%20wanted) are a good starting point.
```
Comment (raboof) [inline README.md]: nice new url

**Comment by ktoso** [comment]
I've just PRed https://github.com/akka/akka/pull/23802 in akka btw

**Comment by ktoso** [comment]
No need to wait on validation :P

**Comment by jrudolph** [comment]
Btw. do you know if github features this label somewhere?

**Comment by ktoso** [comment]
I was actually looking today and did not find anything, they did not put it on the blog either yet, just saw the popup when browsing tickets... Perhaps that's what they'll do? show such popup? But I've not seen anything yet.

**Comment by ktoso** [comment]
Kind of being led around by the nose by github here but likely good to match their recommendation

**Pull request merged**